packagelist

```diff
@@ -0,0 +1,7 @@
+jq
+wget
+git
+gawk
+python3-minimal
+python3-pip
+unzip
```

Comment:
you could probably get around using awk. It would also be nice to add a comment explaining why each of these are needed if possible.

Reply:
It's not possible because that's passed as an argument directly into the apt-get call in the Dockerfile

Reply:
However I'll add it to the list in the README

Reply:
you could do some pre processing, like `grep -v "#"`